### PR TITLE
Task 1237

### PR DESCRIPTION
**Pull request by ahmedelsayed-93**
fixes https://github.com/zero-os/0-orchestrator/issues/1237

### CODE REVIEW

**Comment by ahmedelsayed-93** [comment]
```
 % nosetests -s -v testcases/basic_tests/test05_storagepools_apis.py:TestStoragepoolsAPI.test017_post_storagepool_filesystem_snapshot_rollback  --tc-file config.ini
GAT-152 ... ok
----------------------------------------------------------------------
Ran 1 test in 105.771s

OK
```

File: tests/0_orchestrator/test_suite/testcases/basic_tests/test05_storagepools_apis.py [hunk 1]
```diff
@@ -481,8 +482,87 @@ def test016_delete_storagepool_filesystem_snapshot(self):
...
+        filesystem_path = '/mnt/storagepools/{}/filesystems/{}'.format(
+            self.data['name'], self.data_filesystem['name']
+        )
+
+        self.lg.info("Create file test.txt on filesystem (FS0)")
+        cmd = 'echo "test" > {}/test.txt'.format(filesystem_path)
+        response = self.core0_client.client.bash(cmd).get()
+        self.assertEqual(response.state, 'SUCCESS')
+
+        self.lg.info("Make sure file test.txt is created")
```
Comment (john-kheir) [inline tests/0_orchestrator/test_suite/testcases/basic_tests/test05_storagepools_apis.py]: remove this check

File: tests/0_orchestrator/test_suite/framework/orchestrator_apis/storagepools_apis.py [hunk 2]
```diff
@@ -104,9 +104,9 @@ def delete_filesystem_snapshots_snapshotname(self, nodeid, storagepoolname, file
                                                                        filesystemname=filesystemname,
                                                                        snapshotname=snapshotname)
 
-    @catch_exception_decoration_return
+    @catch_exception_decoration
     def post_filesystem_snapshots_snapshotname_rollback(self, nodeid, storagepoolname, filesystemname, snapshotname,
-                                                        data):
+                                                        data={}):
```
Comment (islamTaha12) [inline tests/0_orchestrator/test_suite/framework/orchestrator_apis/storagepools_apis.py]: what is the value of this data?

Comment (john-kheir) [reply tests/0_orchestrator/test_suite/framework/orchestrator_apis/storagepools_apis.py]: i think the data should be remove

Comment (ahmedelsayed-93) [reply tests/0_orchestrator/test_suite/framework/orchestrator_apis/storagepools_apis.py]: the python client rollback function description
```
Type:        method
String form: <bound method NodesService.RollbackFilesystemSnapshot of <zeroos.orchestrator.client.nodes_service.NodesService object at 0x7fd3e8a47128>>
File:        /usr/local/lib/python3.5/dist-packages/zeroos/orchestrator/client/nodes_service.py
Definition:  c.nodes.RollbackFilesystemSnapshot(self, data, snapshotname, filesystemname, storagepoolname, nodeid, headers=None, query_params=None, content_type='application/json')
Source:
    def RollbackFilesystemSnapshot(self, data, snapshotname, filesystemname, storagepoolname, nodeid, headers=None, query_params=None, content_type="application/json"):
        """
        Rollback the file system to the state at the moment the snapshot was taken
        It is method for POST /nodes/{nodeid}/storagepools/{storagepoolname}/filesystems/{filesystemname}/snapshots/{snapshotname}/rollback
        """
        uri = self.client.base_url + "/nodes/"+nodeid+"/storagepools/"+storagepoolname+"/filesystems/"+filesystemname+"/snapshots/"+snapshotname+"/rollback"
        return self.client.post(uri, data, headers, query_params, content_type)

```
it expects to receive data parameter and i filed and issue in go-raml and it is solved but may be the client is generated using an old version, so to work around that i set the data parameter to {}

Comment (islamTaha12) [reply tests/0_orchestrator/test_suite/framework/orchestrator_apis/storagepools_apis.py]: so u dont have to pass it as a method parameter.

**Pull request merged**